Add `The Roundtable Open @ LVL UP EXPO` news post (#13058)
ppy/osu-wiki: news/2025/2025-03-31-the-roundtable-open-at-lvl-up-expo.md

Ready to go? Here's what you need to know:

## Where and when is this happening?

**The Roundtable Open at LVL UP EXPO** is in Las Vegas, Nevada in the United States.

The expo takes place in the [Las Vegas Convention Center](https://www.google.com/maps/place/Las+Vegas+Convention+Center,+South+Hall/@36.1286087,-115.1541175,878m/data=!3m2!1e3!4b1!4m6!3m5!1s0x80c8c46640af22e7:0xa93a4afe2fe7b046!8m2!3d36.1286087!4d-115.1515426!16zL20vMDU4eDJi) (South Hall - 3150 Paradise Rd, Las Vegas, NV 89109). On the **2nd floor**, walk towards the **Game Hall, Main Events, & Panels** space. From here, look for the **PC Gaming** and **PC Stage** area. You'll see some familiar pink circles at this point.

![floor 2 map](/wiki/shared/news/2025-03-31-the-roundtable-open-at-lvl-up-expo/floor_2_map.jpg)

The event is **3 days**, lasting from **April 25th to April 27th**. A valid badge for the convention is required for attendance. Get your ticket on [the event's website](https://lvlupexpo.com/) and use code `OSULVLUP25` for $10 off a 3-day badge!

### Expo schedule

| Date | Day | Hours |
| --: | :-- | :-- |

> | April 25 | Friday | 12:00PM to 8:00PM |
> | April 26 | Saturday | 11:00AM to 8:00PM |
> | April 27 | Sunday | 11:00AM to 6:00PM |

Reviewer: ```suggestion
| April 25 | Friday | 12:00 PM to 8:00 PM |
| April 26 | Saturday | 11:00 AM to 8:00 PM |
| April 27 | Sunday | 11:00 AM to 6:00 PM |
```

Mappools for both divisions will be revealed during the event!

#### Division 1

| Stage | Day | Notes |
| --: | :-- | :-- |
| **Registration** | Before qualifiers finish | [Register here](https://www.start.gg/tournament/the-roundtable-x-lvlup-expo-division-1/details) or at the event! This division is intended for the most competitive players. |
| **Qualifiers** | Friday | Qualifiers lobbies run hourly. After checking in on-site, you'll be assigned to a qualifiers lobby. Top 32 players from qualifiers will move onto the bracket stage. |
| **Round of 32** | Saturday | Best of 7, 1 ban |
| **Round of 16** | Saturday | Best of 7, 1 ban |
| **Quarterfinals** | Saturday | Best of 9, 1 ban |
| **Semifinals** | Saturday | Best of 9, 1 ban |
| **Finals** | Saturday | Best of 9, 2 bans |
| **Grand Finals** | Saturday | Best of 11, 2 bans |

#### Division 2

| Stage | Day | Notes |
| --: | :-- | :-- |
| **Registration** | Before qualifiers finish | [Register here](https://www.start.gg/tournament/the-roundtable-x-lvlup-expo-division-2/details) or at the event! This division is intended for semi-casual players who want to experience osu! in a competitive setting. |

> | **Qualifiers** | Saturday | Qualifiers lobbies run hourly. After checking in on-site, you'll be assigned to a qualifiers lobby. Top 16 players from qualifiers will move onto the bracket stage. |

Reviewer: ```suggestion
| **Qualifiers** | Saturday | Qualifier lobbies run hourly. After checking in on-site, you'll be assigned to a qualifier lobby. The top 16 players from the qualifiers will move onto the bracket stage. |
```

| **Round of 16** | Sunday | Best of 7, 1 ban |
| **Quarterfinals** | Sunday | Best of 9, 1 ban |
| **Semifinals** | Sunday | Best of 9, 1 ban |
| **Finals** | Sunday | Best of 9, 2 bans |
| **Grand Finals** | Sunday | Best of 11, 2 bans |

## Can I help out?

> If your dream is to wrangle osu! players to their designated PCs, you can [apply to be a volunteer for the tournament here!](https://x.com/TheRoundtableLA/status/1900661907608268990) If accepted, you'll receive a 3-day volunteer badge at no cost.

Reviewer: would be convenient to not have to click twice and wait for page loads or be interrupted by twitter login walls

```suggestion
If your dream is to wrangle osu! players to their designated PCs, you can [apply to be a volunteer for the tournament here!](https://docs.google.com/forms/d/e/1FAIpQLSex7f1YZfbUQbmR8gdb7_7dy-rNFk0Tz7Y6o7s04V7h7S0ooQ/viewform) If accepted, you'll receive a 3-day volunteer badge at no cost.
```

---

> Mark your calendars: **April 25th** to **April 27th** are the dates for the first iteration of **The Roundtable Open**. Grab your tickets from [LVL UP EXPO's website](https://lvlupexpo.com/) with code `OSULVLUP25` for $10 off a 3-day badge.

Reviewer: does the promo code need to be promoted twice
